tardis/montecarlo/tests/Makefile

```diff
@@ -0,0 +1,22 @@
+all: test_cmontecarlo
+
+rk_isaac.o: ../src/randomkit/rk_isaac.c
+	g++ -fpermissive -c ../src/randomkit/rk_isaac.c
```

Comment:
The make file is only for linux. Take a look at my `setup_package.py` file. I directly built them as an extension.